connector/src/main/java/org.apache.flink/connector/nebula/utils/WriteModeEnum.java

```diff
@@ -26,4 +26,19 @@ public enum WriteModeEnum {
     WriteModeEnum(String mode) {
         this.mode = mode;
     }
+
+    public static boolean checkValidWriteMode(String modeName) {
+        return chooseWriteMode(modeName) != INSERT
+                || INSERT.name().equalsIgnoreCase(modeName);
```

Comment:
what's the purpose for this function?  always return true?

Reply:
This function is going to be used to verify the `write-mode` parameter passed in through `with` in flink sql. This Function returns true if the mode parameter is one of INSERT, UPDATE, DELETE(ignoreCase), return false if it is any other string, such `'write-mode' = 'add'`.

```java
public static boolean checkValidWriteMode(String modeName) {
    return chooseWriteMode(modeName) != INSERT
            || INSERT.name().equalsIgnoreCase(modeName);
}

public static WriteModeEnum chooseWriteMode(String modeName) {
    if (UPDATE.name().equalsIgnoreCase(modeName)) {
        return UPDATE;
    }
    if (DELETE.name().equalsIgnoreCase(modeName)) {
        return DELETE;
    }
    return INSERT;
}
```

But now the `write-mode` has been removed from with clause, and it seems better to use the `enumType` to validate that comes with flink, so this part of the logic can be removed.